src/components/profile/VerifyEditCode.js

```diff
@@ -0,0 +1,184 @@
+// @flow
+import React, { useEffect, useState } from 'react'
+import _get from 'lodash/get'
+import logger from '../../lib/logger/pino-logger'
+import API from '../../lib/API/api'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import { withStyles } from '../../lib/styles'
+import SMSFormComponent from '../signup/SMSFormComponent'
+import { useDialog } from '../../lib/undux/utils/dialog'
+import userStorage from '../../lib/gundb/UserStorage'
+
+const log = logger.child({ from: 'SmsForm' })
+
+export type SMSRecord = {
+  smsValidated: boolean,
+  sentSMS?: boolean,
+}
+
+const NumInputs: number = 6
+
+const VerifyEditCode = props => {
```

Comment:
can a user press "back" in this screen? in case he doesnt get the verificaition email/sms?
"back" should be same as "cancel" in previous screen

Reply:
@sirpy yes, user can click back is he wants